hashes/src/sha256t.rs

```diff
@@ -266,7 +312,7 @@ macro_rules! sha256t_hash_newtype {
 // Workaround macros being unavailable in attributes.
```

Comment:
BTW this is no longer needed in 1.63 and we can just move this to the inside of the other macro. (Maybe this was also available in 1.56, IDK.)

Reply:
Will do as follow up.